zap/src/main/java/org/parosproxy/paros/control/Control.java

```diff
@@ -666,4 +671,43 @@ public void setMode(Mode mode) {
             model.getOptionsParam().getViewParam().setMode(mode.name());
         }
     }
+
+    public static List<String> getRecentSessionList() {
+        if (recentSessionList == null) {
+            recentSessionList =
+                    new LinkedList<>(
+                            Arrays.asList(
+                                    Model.getSingleton()
+                                            .getOptionsParam()
+                                            .getConfig()
+                                            .getStringArray("recentsessions.path")));
```

Comment:
This should be part of `OptionsParamView` and be under the `view` key, that's where UI state/options are kept.

Reply:
Just to clarify, all of the changes to `Control` should be relocated to `OptionsParamView` :wink:

Reply:
To be clear the `recentSessionList` should be kept there as well.

Reply:
Sorry for the delay, I was busy this month.